### PR TITLE
refactor: Refactor code to start using go-github library to get reviewers list.

### PR DESCRIPTION
**Pull request by deekoder**

### CODE REVIEW

File: github-get-reviewers.go [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+var rStatus []ReviewStatus
+
+func getReviewers(repo string, number int) []ReviewStatus {
+	rStatus = nil
+	// list all reviewers to an assigned PR # in a repo
+	rList, _, err := buzzClient.PullRequests.ListReviewers(ctx, "minio", repo, number)
+	if err != nil {
+		//We should not exit on error in this case.
+		fmt.Println("Unable to get Reviewers for %s", number)
```
Comment (harshavardhana) [inline github-get-reviewers.go]: You don't want to return right here?

Comment (harshavardhana) [reply github-get-reviewers.go]: BTW comments "//a" should be "// a"

**Review by harshavardhana — CHANGES_REQUESTED** [review]

File: github-get-reviewers.go (hunk 1 above)
Comment (deekoder) [reply github-get-reviewers.go]: yes @harshavardhana  i dont want to return. I will fix the space issue.

Comment (deekoder) [reply github-get-reviewers.go]: fixed

File: github-get-reviewers.go [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+package main
+
+import "fmt"
+
+// ReviewStatus - holds Review info
+type ReviewStatus struct {
+	User struct {
+		Login string `json:"login"`
+	} `json:"user"`
+	//State string `json:"state"`
```
Comment (harshavardhana) [inline github-get-reviewers.go]: One more comment missing.. the spacing.

Comment (deekoder) [reply github-get-reviewers.go]: that is a json struct variable i will be using later. Anyway removed for now.

**Review by harshavardhana — APPROVED** [review]

**Pull request merged**